test/sql-features/mzcompose.py

```diff
@@ -0,0 +1,225 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+import argparse
+import os
+import tempfile
+from textwrap import dedent, indent
+
+from materialize.mzcompose import Composition, WorkflowArgumentParser
+from materialize.mzcompose.services import Materialized, Testdrive
+
+SERVICES = [
+    Materialized(unsafe_mode=False),
+    Testdrive(no_reset=True, seed=1),
+]
+
+MZ_SYSTEM_CONNECTION_URL = (
+    "postgres://mz_system:materialize@${testdrive.materialize-internal-sql-addr}"
+)
+USER_CONNECTION_URL = (
+    "postgres://materialize:materialize@${testdrive.materialize-sql-addr}"
+)
+
+
+def header(test_name: str, drop_schema: bool) -> str:
+    """Generate a TD header for a SQL feature test scenario."""
+    header = dedent(
+        f"""
+        # Feature test for SQL feature test: {test_name}
+        #####################################{'#' * len(test_name)}
+        """
+    )
+    # Re-create schema (optional).
+    if drop_schema:
```

Comment:
Technically speaking you do not need to do that -- just never drop the schema throughout the test.

Reply:
If we have multiple test scenarios it might be better to start each scenario on a clean slate.